rdkcentral/Lightning-UI-Components
# fix(Keyboard): make row render when key is empty

## Description
The ticket initially was regarding a "gap" within the keyboard and was to fix so that all the rows would shift up to fill this row gap. Upon further investigation, it was found that this was not an intentional gap. There was a row that was suppose to be there that was failing to render due to the keyboard not returning `type: Key` in the items array of the row when the key is `''`. This fix was to create a "key" to fill this space but mark it as skipFocus and have an alpha value of 0.01. This allows all keys to be rendered and lined up correctly in the row.

## References

LUI-1319

## Testing
- migrate to keyboard-qwerty
- click the key labeled `áöû` (accents)
- notice how the fourth row renders correctly now where as prior it was a gap.

## Automation



## Checklist
